Vagrantfile

```diff
@@ -0,0 +1,16 @@
+$provision_script = <<SCRIPT
```

Comment:
This is a little hard to read but I'm not sure there's a better option.  I guess you could put it in a separate file and read it?